charts/stardog/templates/statefulset.yaml

```diff
@@ -27,6 +27,12 @@ spec:
         app.kubernetes.io/version: {{ .Chart.AppVersion }}
         app.kubernetes.io/component: server
     spec:
+{{ if .Values.securityContext.enabled }}
```

Comment:
we use these when testing deploys to our local, non-Openshift k8s. to mimic the OS behaviour of nonroot userid. 
pattern borrowed from a minio chart.